app/models/chef.rb

```diff
@@ -1,4 +1,10 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+
+   has_many :table_ingredients, through: :dishes
+   
+   def list_of_ingredients
+      self.table_ingredients.distinct
```

Comment:
Awesome work here! Great job adding the additional association in order to more easily grab the ingredients.